### ISSUE TITLE
Error for varsel: arguments imply differing number of rows: 76, 77

### ISSUE THREAD

**Issue by tmalsburg** [stan-dev/projpred#180]
Following [this vignette](https://mc-stan.org/projpred/articles/quickstart_glmm.html) I'm running this code:

``` R
stan_glm(Y ~ V1 + V2 + V3 + V4 + V5 + V6 + V7 + V8 + V9, chains = 4, data=d, seed=1) -> fit

get_refmodel(fit) -> ref
varsel(ref) -> vs
```

but I get this error message:

```
Error in data.frame(.z = y, data) : 
  arguments imply differing number of rows: 76, 77
```

`dim(d)` is `77 11`.  However, the same works with a different but similar data set.  Happy to share the data privately.

**Comment by AlejandroCatalina** [comment]
Is it possible for one row to contain a NaN value or something similar? I can inspect this case personally if you share the data! Thanks for reporting!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Titus von der Malsburg ***@***.***>
Sent: Wednesday, July 21, 2021 3:38:42 PM
To: stan-dev/projpred ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [stan-dev/projpred] Error for varsel: arguments imply differing number of rows: 76, 77 (#180)


Following this vignette<https://mc-stan.org/projpred/articles/quickstart_glmm.html> I'm running this code:

stan_glm(Y ~ V1 + V2 + V3 + V4 + V5 + V6 + V7 + V8 + V9, chains = 4, data=d, seed=1) -> fit

get_refmodel(fit) -> ref
varsel(ref) -> vs

but I get this error message:

Error in data.frame(.z = y, data) :
  arguments imply differing number of rows: 76, 77


dim(d) is 77 11. However, the same works with a different but similar data set. Happy to share the data privately.

**Comment by tmalsburg** [comment]
Yes, there were NAs. I suppose it would make sense to have some checking in the package. Either throw an error or somehow filter the data for complete cases.

**Comment by AlejandroCatalina** [comment]
To be honest I’m not sure that’s the issue, but it might be if some of the underlying functions handle these differently. I’ll check!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by fweber144** [comment]
Should be fixed in `develop` by #274.

**Comment by tmalsburg** [comment]
Tested and it works now.  Thank you!
